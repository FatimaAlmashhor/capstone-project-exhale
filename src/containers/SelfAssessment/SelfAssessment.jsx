import React, { useState } from 'react';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import ProgressBar from '../../components/ProgressBar';
import Tasks from '../../services/selfAssessmentQuestion';
import SelfAssessmentPanel from '../../components/SelfAssessmentPanle';

const SelfAssessment = () => {
  const [result, setResult] = useState([]);
  const [displayIndex, setDisplayIndex] = useState(0);

  const display = (index) => (index === displayIndex ? 'block' : 'hidden');

  const handleClick = (e) => {
    // inset the result of answer
    if (displayIndex === Tasks.length - 1) {
      // fetch the the result
    }
    setResult((pre) => [...pre], (result[displayIndex] = e.target.dataset.rate));
    setDisplayIndex((pre) => (displayIndex === Tasks.length ? pre : pre + 1)); // move to next question
  };
  const handleBackStep = () =>
    setDisplayIndex((pre) => (displayIndex === 0 ? pre : pre - 1));
  const TasksBicker = Tasks.map((element) => (
    <div key={element.id} className={`${display(element.id)}`}>
      <SelfAssessmentPanel task={element.task} onClick={handleClick} />
    </div>
  ));
  return (
    <div className="flex flex-col mx-auto w-4/5 md:w-8/12 lg:w-6/12 mt-10">
      <ProgressBar showLabel max={Tasks.length} now={displayIndex} />
      {TasksBicker}
      <div className="  mt-16  ">
        <button
          type="button"
          className="  self-start focus:outline-none"
          onClick={handleBackStep}
        >
          <FontAwesomeIcon
            className="text-lg md:text-xl pt-1 text-green-400"
            icon={['fas', 'chevron-left']}
          />
          <span className="text-gray-500 text-base ml-5">BACK</span>
        </button>
      </div>
    </div>
  );
};

export default SelfAssessment;
