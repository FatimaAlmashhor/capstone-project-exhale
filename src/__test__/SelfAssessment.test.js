import React from 'react';
import renderer from 'react-test-renderer';
import SelfAssessment from '../containers/SelfAssessment';

it('renders correctly', () => {
  const tree = renderer
    .create(<SelfAssessment /> )
    .toJSON();
  expect(tree).toMatchSnapshot();
});