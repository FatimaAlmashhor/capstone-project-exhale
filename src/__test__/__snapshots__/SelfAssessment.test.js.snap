// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="flex flex-col mx-auto w-4/5 md:w-8/12 lg:w-6/12 mt-10"
>
  <div
    className="relative pt-1"
  >
    <p>
      0\\10
    </p>
    <div
      className="overflow-hidden  h-2 mb-4 text-xs flex rounded bg-blue-200"
    >
      <div
        className="shadow-none  flex flex-col text-center whitespace-nowrap text-white justify-center  bg-yellow-500"
        style={
          Object {
            "width": "0%",
          }
        }
      />
    </div>
  </div>
  <div
    className="block"
  >
    <div>
      <div
        className="flex-auto py-10 "
      >
        <h1
          className=" text-4xl  text-center font-sans transition duration-500  "
        >
          This  is a fake test or question1
        </h1>
      </div>
      <div
        className="flex-auto "
      >
        <div
          className=" flex mt-8 mx-auto justify-center transition duration-500"
        >
          <div
            className=" flex flex-col justify-center self-center flex-1"
          >
            <button
              className=" p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="20"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto text-gray-500  mt-3"
            >
              NEVER
            </p>
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="40"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex justify-center self-center flex-1 "
          >
            <button
              className=" p-3  md:p-6 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500"
              data-rate="60"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="80"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex flex-col justify-center self-center  flex-1"
          >
            <button
              className="p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500  "
              data-rate="100"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto mt-3 text-gray-500"
            >
              ALWAYS
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="hidden"
  >
    <div>
      <div
        className="flex-auto py-10 "
      >
        <h1
          className=" text-4xl  text-center font-sans transition duration-500  "
        >
          This  is a fake test or question1
        </h1>
      </div>
      <div
        className="flex-auto "
      >
        <div
          className=" flex mt-8 mx-auto justify-center transition duration-500"
        >
          <div
            className=" flex flex-col justify-center self-center flex-1"
          >
            <button
              className=" p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="20"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto text-gray-500  mt-3"
            >
              NEVER
            </p>
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="40"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex justify-center self-center flex-1 "
          >
            <button
              className=" p-3  md:p-6 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500"
              data-rate="60"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="80"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex flex-col justify-center self-center  flex-1"
          >
            <button
              className="p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500  "
              data-rate="100"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto mt-3 text-gray-500"
            >
              ALWAYS
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="hidden"
  >
    <div>
      <div
        className="flex-auto py-10 "
      >
        <h1
          className=" text-4xl  text-center font-sans transition duration-500  "
        >
          This  is a fake test or question2
        </h1>
      </div>
      <div
        className="flex-auto "
      >
        <div
          className=" flex mt-8 mx-auto justify-center transition duration-500"
        >
          <div
            className=" flex flex-col justify-center self-center flex-1"
          >
            <button
              className=" p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="20"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto text-gray-500  mt-3"
            >
              NEVER
            </p>
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="40"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex justify-center self-center flex-1 "
          >
            <button
              className=" p-3  md:p-6 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500"
              data-rate="60"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="80"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex flex-col justify-center self-center  flex-1"
          >
            <button
              className="p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500  "
              data-rate="100"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto mt-3 text-gray-500"
            >
              ALWAYS
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="hidden"
  >
    <div>
      <div
        className="flex-auto py-10 "
      >
        <h1
          className=" text-4xl  text-center font-sans transition duration-500  "
        >
          This  is a fake test or question3
        </h1>
      </div>
      <div
        className="flex-auto "
      >
        <div
          className=" flex mt-8 mx-auto justify-center transition duration-500"
        >
          <div
            className=" flex flex-col justify-center self-center flex-1"
          >
            <button
              className=" p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="20"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto text-gray-500  mt-3"
            >
              NEVER
            </p>
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="40"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex justify-center self-center flex-1 "
          >
            <button
              className=" p-3  md:p-6 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500"
              data-rate="60"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="80"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex flex-col justify-center self-center  flex-1"
          >
            <button
              className="p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500  "
              data-rate="100"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto mt-3 text-gray-500"
            >
              ALWAYS
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="hidden"
  >
    <div>
      <div
        className="flex-auto py-10 "
      >
        <h1
          className=" text-4xl  text-center font-sans transition duration-500  "
        >
          This  is a fake test or question4
        </h1>
      </div>
      <div
        className="flex-auto "
      >
        <div
          className=" flex mt-8 mx-auto justify-center transition duration-500"
        >
          <div
            className=" flex flex-col justify-center self-center flex-1"
          >
            <button
              className=" p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="20"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto text-gray-500  mt-3"
            >
              NEVER
            </p>
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="40"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex justify-center self-center flex-1 "
          >
            <button
              className=" p-3  md:p-6 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500"
              data-rate="60"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="80"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex flex-col justify-center self-center  flex-1"
          >
            <button
              className="p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500  "
              data-rate="100"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto mt-3 text-gray-500"
            >
              ALWAYS
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="hidden"
  >
    <div>
      <div
        className="flex-auto py-10 "
      >
        <h1
          className=" text-4xl  text-center font-sans transition duration-500  "
        >
          This  is a fake test or question5
        </h1>
      </div>
      <div
        className="flex-auto "
      >
        <div
          className=" flex mt-8 mx-auto justify-center transition duration-500"
        >
          <div
            className=" flex flex-col justify-center self-center flex-1"
          >
            <button
              className=" p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="20"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto text-gray-500  mt-3"
            >
              NEVER
            </p>
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="40"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex justify-center self-center flex-1 "
          >
            <button
              className=" p-3  md:p-6 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500"
              data-rate="60"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="80"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex flex-col justify-center self-center  flex-1"
          >
            <button
              className="p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500  "
              data-rate="100"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto mt-3 text-gray-500"
            >
              ALWAYS
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="hidden"
  >
    <div>
      <div
        className="flex-auto py-10 "
      >
        <h1
          className=" text-4xl  text-center font-sans transition duration-500  "
        >
          This  is a fake test or question6
        </h1>
      </div>
      <div
        className="flex-auto "
      >
        <div
          className=" flex mt-8 mx-auto justify-center transition duration-500"
        >
          <div
            className=" flex flex-col justify-center self-center flex-1"
          >
            <button
              className=" p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="20"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto text-gray-500  mt-3"
            >
              NEVER
            </p>
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="40"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex justify-center self-center flex-1 "
          >
            <button
              className=" p-3  md:p-6 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500"
              data-rate="60"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="80"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex flex-col justify-center self-center  flex-1"
          >
            <button
              className="p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500  "
              data-rate="100"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto mt-3 text-gray-500"
            >
              ALWAYS
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="hidden"
  >
    <div>
      <div
        className="flex-auto py-10 "
      >
        <h1
          className=" text-4xl  text-center font-sans transition duration-500  "
        >
          This  is a fake test or question7
        </h1>
      </div>
      <div
        className="flex-auto "
      >
        <div
          className=" flex mt-8 mx-auto justify-center transition duration-500"
        >
          <div
            className=" flex flex-col justify-center self-center flex-1"
          >
            <button
              className=" p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="20"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto text-gray-500  mt-3"
            >
              NEVER
            </p>
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="40"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex justify-center self-center flex-1 "
          >
            <button
              className=" p-3  md:p-6 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500"
              data-rate="60"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="80"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex flex-col justify-center self-center  flex-1"
          >
            <button
              className="p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500  "
              data-rate="100"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto mt-3 text-gray-500"
            >
              ALWAYS
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="hidden"
  >
    <div>
      <div
        className="flex-auto py-10 "
      >
        <h1
          className=" text-4xl  text-center font-sans transition duration-500  "
        >
          This  is a fake test or question8
        </h1>
      </div>
      <div
        className="flex-auto "
      >
        <div
          className=" flex mt-8 mx-auto justify-center transition duration-500"
        >
          <div
            className=" flex flex-col justify-center self-center flex-1"
          >
            <button
              className=" p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="20"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto text-gray-500  mt-3"
            >
              NEVER
            </p>
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="40"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex justify-center self-center flex-1 "
          >
            <button
              className=" p-3  md:p-6 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500"
              data-rate="60"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="80"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex flex-col justify-center self-center  flex-1"
          >
            <button
              className="p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500  "
              data-rate="100"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto mt-3 text-gray-500"
            >
              ALWAYS
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="hidden"
  >
    <div>
      <div
        className="flex-auto py-10 "
      >
        <h1
          className=" text-4xl  text-center font-sans transition duration-500  "
        >
          This  is a fake test or question9
        </h1>
      </div>
      <div
        className="flex-auto "
      >
        <div
          className=" flex mt-8 mx-auto justify-center transition duration-500"
        >
          <div
            className=" flex flex-col justify-center self-center flex-1"
          >
            <button
              className=" p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="20"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto text-gray-500  mt-3"
            >
              NEVER
            </p>
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="40"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex justify-center self-center flex-1 "
          >
            <button
              className=" p-3  md:p-6 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500"
              data-rate="60"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className="flex justify-center self-center flex-1 "
          >
            <button
              className=" p-4  md:p-8 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500 "
              data-rate="80"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
          </div>
          <div
            className=" flex flex-col justify-center self-center  flex-1"
          >
            <button
              className="p-6  mt-6 mx-auto md:p-10 bg-transparent border-4 border-gray-600 rounded-full focus:outline-none transition duration-500  "
              data-rate="100"
              onClick={[Function]}
              onMouseEnter={[Function]}
              type="button"
            />
            <p
              className="mx-auto mt-3 text-gray-500"
            >
              ALWAYS
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="  mt-16  "
  >
    <button
      className="  self-start focus:outline-none"
      onClick={[Function]}
      type="button"
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-chevron-left fa-w-10 text-lg md:text-xl pt-1 text-green-400"
        data-icon="chevron-left"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={Object {}}
        viewBox="0 0 320 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M34.52 239.03L228.87 44.69c9.37-9.37 24.57-9.37 33.94 0l22.67 22.67c9.36 9.36 9.37 24.52.04 33.9L131.49 256l154.02 154.75c9.34 9.38 9.32 24.54-.04 33.9l-22.67 22.67c-9.37 9.37-24.57 9.37-33.94 0L34.52 272.97c-9.37-9.37-9.37-24.57 0-33.94z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
      <span
        className="text-gray-500 text-base ml-5"
      >
        BACK
      </span>
    </button>
  </div>
</div>
`;
